packages/labs/test/index.ts

```diff
@@ -2,4 +2,11 @@
  * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
  */
 
+import "es6-shim";
```

Comment:
wasn't this supposed to be removed?

Reply:
(though it's just test code code, so... yolo)

Reply:
no, you _explicitly_ need to bring it in to support React 16. We're definitely not _removing_ es6-shim from the page runtime; we are pushing the responsibility down to consumers of the library (e.g. the test environment, table-dev-app, docs-app, our other apps, etc.)

Reply:
oh i see